tests/unit/helpers/draftPay.test.js

```diff
@@ -583,6 +583,56 @@ describe('getPaidTransportInfo', () => {
     sinon.assert.calledOnceWithExactly(getTransportInfo, [], 'tamalou', 'jébobolà', 'driving', event.company);
   });
 
+  it('should compute specific transport mode if exist', async () => {
+    const event = {
+      hasFixedService: false,
+      type: 'intervention',
+      auxiliary: {
+        administrative: { transportInvoice: { transportType: 'private' } },
+      },
+      address: { fullAddress: 'jébobolà' },
+      company: new ObjectID(),
+      transportMode: 'public',
+    };
+    const prevEvent = {
+      hasFixedService: false,
+      type: 'intervention',
+      startDate: '2019-01-18T15:46:30.636Z',
+      address: { fullAddress: 'tamalou' },
+    };
+    getTransportInfo.resolves({ distance: 10, duration: 40 });
+    const result = await DraftPayHelper.getPaidTransportInfo(event, prevEvent, []);
+
+    expect(result).toBeDefined();
+    sinon.assert.calledOnceWithExactly(getTransportInfo, [], 'tamalou', 'jébobolà', 'public', event.company);
+  });
+
+  it('should not paid transport if transportMode is not personal car', async () => {
```

Comment:
est ce que le cas de mode de transport par defaut = voiture et mode de transport specifique = transit est testé quelques part ?

Reply:
et peux tu renommer `should return transport duration` pour qu'il soit plus explicite sur le cas ou ca se produit ?

Reply:
oui c'est testé ligne 586 + done